pytorch_lightning/loggers/wandb.py

```diff
@@ -246,6 +246,14 @@ def finalize(self, status: str) -> None:
         if self._checkpoint_callback:
             self._scan_and_log_checkpoints(self._checkpoint_callback)
 
+    def finish(self) -> None:
```

Comment:
What's the difference between this, finalize, and close already on the logger base class?

Reply:
yes, that's the reason this PR is still in draft mode. I am not sure what I want to do here yet. Mainly I want to give a user control over ending an experiment manually.

Finalize: method of the loggers mentions this is for ending an experiment, but not actually all loggers implement it this way by closing their experiments completely. Finalize is called by the fit loop at the end of fit but most loggers will allow to continue logging for example during a subsequent .test() call. 

Close: This method exists, but never called by Lightning. Its purpose is unclear to me